classes/user/PKPUserDAO.inc.php

```diff
@@ -201,47 +203,41 @@ function getReviewersForSubmission($contextId, $submissionId, $round) {
 	function getReviewersNotAssignedToSubmission($contextId, $submissionId, &$reviewRound, $name = '') {
 		$reviewAssignmentDao = DAORegistry::getDAO('ReviewAssignmentDAO');
 
-		$params = array((int) $contextId, ROLE_ID_REVIEWER, (int) $reviewRound->getStageId(), (int) $submissionId, (int) $reviewRound->getId());
-		if (!empty($name)) $params[] = $params[] = $params[] = $params[] = "%$name%";
+		$params = array(
+			(int) $contextId,
+			ROLE_ID_REVIEWER,
+			(int) $reviewRound->getStageId(),
+		);
+		$params = array_merge($params, $this->getFetchParameters());
+		$params[] = (int) $submissionId;
+		$params[] = (int) $reviewRound->getId();
+		if (!empty($name)) {
+			$nameSearchJoins = 'LEFT JOIN user_settings usgs ON (u.user_id = usgs.user_id AND usgs.setting_name = \'' . IDENTITY_SETTING_GIVENNAME .'\')
+				LEFT JOIN user_settings usfs ON (u.user_id = usfs.user_id AND usfs.setting_name = \'' . IDENTITY_SETTING_FAMILYNAME .'\')';
+			$params[] = $params[] = $params[] = $params[] = "%$name%";
```

Comment:
Nice, never seen that syntax before! :joy:

Reply:
Hihihihi :-) Yes. It was not my creativity :-) Took me even a while to understand... Hihihihi :-)